Lib/test/test_fstring.py

```diff
@@ -361,6 +361,18 @@ def test_backslashes_in_string_part(self):
         self.assertEqual(f'2\x203', '2 3')
         self.assertEqual(f'\x203', ' 3')
 
+        with self.assertWarns(DeprecationWarning):
+            value = eval(r"f'\{6*7}'")
+        self.assertEqual(value, '\\42')
+        self.assertEqual(f'\\{6*7}', '\\42')
+        self.assertEqual(fr'\{6*7}', '\\42')
+
+        AMPERSAND = 123
```

Comment:
Right. But the important part (to me) that I missed is that it's not equal to `ord('&')`. I had assumed it was, and that somehow that value was being used to produce the '&' in the output. But the point is that the value of the local `AMPERSAND` is **not** being used, correct?

If so, how about using 0, since most people by inspection know it's not an `&`?